inst/include/component_data.hpp

```diff
@@ -65,6 +65,8 @@
 #define D_RHO_OC "rho_oc"            // BC radiative efficiency
 #define D_RHO_SO2 "rho_so2"          // SO2 radiative efficiency
 #define D_RHO_NH3 "rho_nh3"          // NH3 radiative efficiency
+#define D_AERO_SCALE "alpha"
+#define D_VOLCANIC_SCALE "volscl"
```

Comment:
These are both scalars; should they have a consistent naming structure? E.g. `aero_scalar` and `vol_scalar`?